src/core/qgsmaprenderertask.cpp

```diff
@@ -154,6 +156,38 @@ bool QgsMapRendererTask::run()
       mError = ImageSaveFail;
       return false;
     }
+
+    if ( mSaveWorldFile )
+    {
+      QString content;
+      // note: use 17 places of precision for all numbers output
```

Comment:
@nyalldawson , it's a brilliant idea -- I didn't like having this both in qgsmapcanvas.cpp and there.